rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/utils/Zipper.java

```diff
@@ -119,10 +125,14 @@ private static void zipFiles(List<File> files, String basepath, OutputStream os)
                 closer.register(zos);
                 if (files.size() == 1) {
                     File file = files.get(0);
-                    writeZipEntry(new ZipEntry(file.getName()), new FileInputStream(file), zos);
+                    FileInputStream inputStream = new FileInputStream(file);
```

Comment:
don't we need to close this inputStream ?

Reply:
this is done by the "closer"